runtime/parachains/src/reward_points.rs

```diff
@@ -22,42 +22,71 @@
 //! for the time being, although we will build schemes to do so in the future.
 
 use crate::{session_info, shared};
-use frame_support::traits::ValidatorSet;
-use primitives::v2::ValidatorIndex;
+use frame_support::traits::{Defensive, ValidatorSet};
+use primitives::v2::{SessionIndex, ValidatorIndex};
+use sp_std::collections::btree_set::BTreeSet;
 
 /// The amount of era points given by backing a candidate that is included.
 pub const BACKING_POINTS: u32 = 20;
+/// The amount of era points given by dispute voting on a candidate.
+pub const DISPUTE_STATEMENT_POINTS: u32 = 15;
```

Comment:
Curious about how we got to 15, also have the same question for the `BACKING_POINTS`.

Reply:
I didn't pick the `BACKING_POINTS` amount, but it's the same as block production reward.
Disputes are expected to be rare and when they happen everyone participates, so the amount shouldn't matter much I guess? Only those who don't vote (and on the wrong side) will lose when it comes to the staking pot distribution.


And as noted in the description
> the era points amount is open for discussion

Reply:
made them same as backing points as discussed on the call with @eskimor